test/functions/iterators/elliot/report/inline-iters-report.default.good

```diff
@@ -0,0 +1,58 @@
+Inlined iterator (tenYieldIter) in module inline-iters-report (inline-iters-report.chpl:10)
```

Comment:
Is the order of these `--report-inlined-iterators` messages guaranteed, or do you need something like a `.prediff` to sort them?

Reply:
I was wondering the same thing. `functions/iterators/sungeun/report/` (which tests `--report-optimized-loop-iterators`) does not sort and it has been stable for the past 4 years, so I think the ordering is relatively safe. Note that the `--inline-iterators` and `--optimize-loop-iterators` functions are both called from `expandForLoop()`, so this is no less stable than that test.